tests-and-examples/tests/cases/auth_context.rs

```diff
@@ -12,7 +13,7 @@ use tests_and_examples::util::{read_cert_pair, read_single_crt};
 
 #[derive(Clone)]
 struct GreeterService {
-    tx: Sender<(String, String)>,
+    tx: Sender<std::result::Result<Option<HashMap<String, String>>, ()>>,
```

Comment:
Seems `Option<HashMap<K, V>>` is enough.

Reply:
Fixed